keps/sig-storage/3107-csi-nodeexpandsecret/README.md

```diff
@@ -220,53 +237,137 @@ N/A
 
 ### Rollout, Upgrade and Rollback Planning
 
-TBD
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-TBD
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
```

Comment:
Anything around CSI drivers to consider during rollout/rollback? Are there specific rules around how CSI drivers must handle the new field for this to be safe?

Reply:
no, we are safe with rollout/rollback failure. I have updated the section too.